paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsStateFactory.kt

```diff
@@ -104,6 +104,7 @@ private fun List<PaymentOptionsItem>.findSelectedPosition(paymentSelection: Paym
                 }
             }
             is PaymentSelection.New -> false
+            is PaymentSelection.ExternalPaymentMethod -> false
```

Comment:
It looks like this is because we only use this function in a view that shows GooglePay, Link, and saved payment methods. It's kind of confusing to me though